baseplate/sidecars/live_data_watcher.py

```diff
@@ -35,78 +31,23 @@
 HEARTBEAT_INTERVAL = 300
 
 
+class LoaderType(Enum):
+    RAW = 'RAW'
```

Comment:
🔕  "Raw" to me implies some parsing of the data (like mime type), "default"/"zookeeper"/"direct" might be better terminology.